src/libpmemfile-posix/inode.c

```diff
@@ -674,33 +674,35 @@ static int
 _pmemfile_fstatat(PMEMfilepool *pfp, struct pmemfile_vinode *dir,
 		const char *path, struct stat *buf, int flags)
 {
+	int error = 0;
+	struct pmemfile_cred cred;
+	struct pmemfile_path_info info;
+	struct pmemfile_vinode *vinode;
+
+	LOG(LDBG, "path %s", path);
+
 	if (!buf) {
-		errno = EFAULT;
-		return -1;
+		error = EFAULT;
+		goto ini_err;
 	}
 
 	if (path[0] == 0 && (flags & PMEMFILE_AT_EMPTY_PATH)) {
-		LOG(LSUP, "AT_EMPTY_PATH not supported yet");
-		errno = EINVAL;
-		return -1;
+		error = vinode_stat(dir, buf);
+		goto ini_err;
```

Comment:
I'm not sure I get it.
``path[0] == 0`` is not an error when ``PMEMFILE_AT_EMPTY_PATH`` is present, right?
But what?